src/main/scala/fs2/aws/sqs/SQSConsumer.scala

```diff
@@ -0,0 +1,11 @@
+package fs2.aws.sqs
+
+import com.amazon.sqs.javamessaging.SQSConnection
+import javax.jms.MessageListener
+
+trait SQSConsumer[F[_]] {
+  def callback: MessageListener
+  def startConsumer()
+  def shutdown()
```

Comment:
should start and shutdown be in context of F?

Reply:
```def serve[A](stream: fs2.Stream[F, A])(implicit F: Effect[F]): fs2.Stream[F, A] = {
    fs2.Stream
      .bracket(start)(con => F.delay(con.shutdown()))
      .flatMap(con => fs2.Stream.eval(F.delay(con.startConsumer())).drain ++ stream)
  }``` both Shutdown and statConsumer are executed in F context